llvm/lib/Transforms/Instrumentation/PGOInstrumentation.cpp

```diff
@@ -1980,8 +1995,11 @@ PreservedAnalyses PGOInstrumentationGen::run(Module &M,
   auto LookupBFI = [&FAM](Function &F) {
     return &FAM.getResult<BlockFrequencyAnalysis>(F);
   };
+  auto LookupLI = [&FAM](Function &F) {
+    return &FAM.getResult<LoopAnalysis>(F);
+  };
```

Comment:
Honestly I'm not sure why we don't simply pass the `FAM` rather than these lambdas, but this probably shouldn't block your work.